packages/orderbook/test/js_api/gui.test.ts

```
@@ -943,6 +944,57 @@ ${dotrainWithoutVaultIds}
+				);
+			mockServer
+				.forPost('/rpc-url')
+				.once()
+				.withBodyIncluding('0x82ad56cb')
+				.thenSendJsonRpcResult(
+					'0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000001a000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000754656b656e203200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000025432000000000000000000000000000000000000000000000000000000000000'
+				);
+			dotrain3 = `${guiConfig}
+
+${dotrain}`;
+			let gui = await DotrainOrderGui.chooseDeployment(dotrain3, 'some-deployment');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'gui' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)